README.md

```diff
@@ -149,14 +149,14 @@ Give a ⭐️ if you like this project!
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-Microverse materials and Ruby tutorial at <a href="https://www.tutorialspoint.com/ruby-on-rails/rails-installation.htm">
+The design used for this webapp was created by [Gregoire Vella](https://www.behance.net/gregoirevella) and is based on his [Snapscan](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) design I would like to thank [Microverse](https://github.com/microverseinc) for the immense support.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Great work so far. However, I noticed that you didn't deploy the application. Kindly deploy your application because it is a major requirement in this project. If _**Render**_ isn't working, you can use other hosting applications to deploy the application. You can read this [article](https://medium.com/@ricardo_valtierra/10-heroku-alternatives-you-can-use-today-to-deploy-rails-in-paas-8b7fece3e93b) to see other alternatives. 👍 💻